### PR TITLE
fix(contributing-guide): Update guidelines

### LINKED ISSUE #10222: Further Clarification could be used when submitting an issue
While fixing issues, I've noticed that adding the challenge alone helps in identifying where to find the issue in the actual git repository, but I think adding the section the challenge is in would also help. i.e. 

> In the Basic JavaScript section of Front End Development Certification

is more helpful than 

> In the Basic JavaScript section

Thoughts on this?

### PR DESCRIPTION
**Pull request by raisedadead**
Closes #10222

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -178,12 +173,14 @@ accepted by Free Code Camp proper.
...
 
-There are two methods of creating a Pull for Free Code Camp:
-
--   Editing files via the GitHub Interface
--   Editing files on a local clone
+The best way to find out any code, you wish to change/add or remove is using 
+the GitHub search bar at the top of the repository page. For example, you could 
+search for a challenge name and the results will display all the files along 
+with line nos. Then you can proceed to the files and verify this is the area 
+that you were looking forward to edit. Always feel free to reachout to the chat
```
Comment (erictleung) [inline CONTRIBUTING.md]: "Reach out" is two words.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -178,12 +173,14 @@ accepted by Free Code Camp proper.
...
+room when you are not certain of any thing specific in the code.
```
Comment (erictleung) [inline CONTRIBUTING.md]: Should we add in a link to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) room here?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -178,12 +173,14 @@ accepted by Free Code Camp proper.
...
 related issues on our Help Contributors Chat Room
 
-#### Methods
+#### How to find the code in the Free Code Camp codebase to fix/edit?
 
-There are two methods of creating a Pull for Free Code Camp:
-
--   Editing files via the GitHub Interface
--   Editing files on a local clone
+The best way to find out any code, you wish to change/add or remove is using
```
Comment (erictleung) [inline CONTRIBUTING.md]: Should there be a comma here? Maybe update it to

> The best way to find any code you wish to change/add or ...

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -178,12 +173,14 @@ accepted by Free Code Camp proper.
...
+#### How to find the code in the Free Code Camp codebase to fix/edit?
 
-There are two methods of creating a Pull for Free Code Camp:
-
--   Editing files via the GitHub Interface
--   Editing files on a local clone
+The best way to find out any code, you wish to change/add or remove is using 
+the GitHub search bar at the top of the repository page. For example, you could 
+search for a challenge name and the results will display all the files along 
+with line nos. Then you can proceed to the files and verify this is the area
```
Comment (erictleung) [inline CONTRIBUTING.md]: Maybe spell out "line nos" to "line numbers".

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -193,6 +190,13 @@ files. This is critical, because if your PR is not accepted, your copy of
 staging will be forever sullied and the only way to fix it is to delete your
 fork and re-fork.
 
+#### Methods
+
+There are two methods of creating a Pull for Free Code Camp:
```
Comment (erictleung) [inline CONTRIBUTING.md]: "...creating a Pull..." --> "...creating a pull request..."

File: CONTRIBUTING.md (hunk 2 above)
Comment (raisedadead) [inline CONTRIBUTING.md]: I left it intentionally, the links are given at two separate places.

**Comment by raisedadead** [comment]
Thanks for the review @erictleung, mind checking this?

**Comment by erictleung** [comment]
@raisedadead LGTM! 👍  Thanks!

**Pull request merged**